docs/CONTRIBUTING.md

```diff
@@ -94,6 +94,115 @@ Now force push to update your PR:
 git push --force
 ```
 
+## API tests
+
+There are two kinds of tests in the API: unit (`phpunit`) and integration tests (`behat`).
```

Comment:
To be consistent please use one of the two suggestion below:
- `There are two kinds of tests in the API: unit tests (`phpunit`) and integration tests (`behat`).`
- `There are two kinds of tests in the API: unit (`phpunit`) and integration (`behat`) tests.`

2nd seems better.